guide/14-deep-learning/object-detection.ipynb

```diff
@@ -639,12 +639,8 @@
     "\n",
```

Comment:
<p>Delete the phrase 'For&nbsp;<strong>arcgis.learn models</strong> please&nbsp;'. Start the sentence with "Refer to ..." and add a markdown link to the page getting rid of the URL in the rendered cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>